protobuf/flow/entities/network_parameters.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
+
+package entities;
+
+message NetworkParameters {
```

Comment:
Sorry to be difficult, but I don't think `NetworkParameters` needs to be its own entity. Up until now we've been reserving entities for "items that we hash" and they are directly correlated with the models that we store.

I'd prefer to just have each network param be a field in `GetNetworkParametersResponse`.

The table you added above is great, though! I would just move it up to the documentation for `GetNetworkParameters`

Reply:
ok will change it

Reply:
> Sorry to be difficult, but I don't think `NetworkParameters` needs to be its own entity. Up until now we've been reserving entities for "items that we hash" and they are directly correlated with the models that we store.
> 
> I'd prefer to just have each network param be a field in `GetNetworkParametersResponse`.
> 
> The table you added above is great, though! I would just move it up to the documentation for `GetNetworkParameters`

and you are not being difficult :)